aries-site/src/examples/components/table/TableSummaryExample.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Box, DataTable, Heading, ResponsiveContext, Text } from 'grommet';
+
+const data = [
+  {
+    id: 0,
+    application: 'General Purpose VMs',
+    requests: 742,
+    delivered: 623,
+  },
+  {
+    id: 1,
+    application: 'Containers',
+    requests: 23,
+    delivered: 12,
+  },
+  {
+    id: 2,
+    application: 'ML Ops',
+    requests: 42,
+    delivered: 32,
+  },
+  {
+    id: 3,
+    application: 'Private Cloud VMs',
+    requests: 654,
+    delivered: 516,
+  },
+  {
+    id: 4,
+    application: 'Storage Optimized Compute',
+    requests: 298,
+    delivered: 290,
+  },
+  {
+    id: 5,
+    application: 'Hyperconverged VMs',
+    requests: 1074,
+    delivered: 1052,
+  },
+  {
+    id: 6,
+    application: 'Mission Critical Storage',
+    requests: 322,
+    delivered: 273,
+  },
+  {
+    id: 7,
+    application: 'Data Protection',
+    requests: 742,
+    delivered: 623,
+  },
+  {
+    id: 8,
+    application: 'Composable Compute',
+    requests: 33,
+    delivered: 22,
+  },
+  {
+    id: 9,
+    application: 'General Compute',
+    requests: 52,
+    delivered: 32,
+  },
+  {
+    id: 10,
+    application: 'Business Critical Storage',
+    requests: 694,
+    delivered: 616,
+  },
+  {
+    id: 12,
+    application: 'Enterprise-Ready VMs',
+    requests: 1074,
+    delivered: 752,
+  },
+];
+
+const enhancedData = data.map(datum => {
+  const adjustedDatum = { ...datum };
+  adjustedDatum.attachment = datum.delivered / datum.requests;
+  return adjustedDatum;
+});
+
+const columns = [
+  {
+    property: 'requests',
+    header: 'Requests',
+    render: datum => `${datum.requests.toLocaleString()}`,
+    align: 'end',
+    aggregate: 'sum',
+    footer: { aggregate: true },
+  },
+  {
+    property: 'delivered',
+    header: 'Delivered',
+    render: datum => `${datum.delivered.toLocaleString()}`,
+    align: 'end',
+    aggregate: 'sum',
+    footer: { aggregate: true },
+  },
+  {
+    property: 'attachment',
+    header: (
+      <>
+        Attachement&nbsp;
+        <Text color="text" weight="normal">
+          %
+        </Text>
+      </>
+    ),
+    render: datum => `${(datum.attachment * 100).toFixed([1])}`,
+    align: 'end',
+    aggregate: 'avg',
+    footer: { aggregate: true },
+  },
+];
+
+export const TableSummaryExample = () => {
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <>
+      <Heading level={3} margin={{ bottom: 'small', top: 'none' }}>
+        Service Adoption
```

Comment:
Everything looks great. Only remaining item is that on Safari, this Footer still isn't stay fixed to the bottom.

Also, the scroll region is a little funky where I can put my mouse to the right side of the table (in the open white space of the example) and scroll the table from over there. I would expect the scroll region would be confined to the table width.

Reply:
I'm noticing the fixed footer might be a grommet bug (when I open this storybook example on safari, the footer is not remaining fixed http://localhost:9001/?path=/story/datatable--fill-and-pin) -- so don't worry about resolving that here, but just take a look at the scroll region.

Reply:
I fixed the scroll region to align with the DataTable edges.

I believe the fixed footer needs to be resolved in Grommet. View [this story](https://storybook.grommet.io/?path=/story/datatable--fill-and-pin) on Safari and you will see the same behavior.

It looks like `position: sticky; bottom: 0;` is being set on the `<td />`'s whereas it looks like it needs to be set on `<tfooter />`. From what I remember, this is implemented in how pin is being treated with context. This may need to be rearchitected. I will create an issue to represent.

Reply:
Cool, thanks for creating an issue to track.